ocp_resources/resource.py

```diff
@@ -317,6 +317,9 @@ class ApiGroup:
         V2V_KUBEVIRT_IO = "v2v.kubevirt.io"
         VELERO_IO = "velero.io"
         VM_KUBEVIRT_IO = "vm.kubevirt.io"
+        OBSERVABILITY_ADDON_IO = (
```

Comment:
use `OBSERVABILITY_OPEN_CLUSTER_MANAGEMENT_IO` in f-string

Reply:
done